### ISSUE TITLE
Limited support for $filter and $orderby with delta. Plans to remove limitation?

### ISSUE THREAD

**Issue by mishakogan** [microsoftgraph/microsoft-graph-docs#2820]
Issue: Limited support for $filter and $orderby

> OData query parameters
> You can use a $select query parameter as in any GET request to specify only the properties your need for best performance. The  id property is always returned.
> Delta query support $select, $top, and $expand for messages.
> There is limited support for $filter and $orderby:
> The only supported $filter expresssions are $filter=receivedDateTime+ge+{value} or $filter=receivedDateTime+gt+{value}.
> The only supported $orderby expression is $orderby=receivedDateTime+desc. If you do not include an $orderby expression, the return order is not guaranteed.
> There is no support for $search.

^ Are there any plans to remove limitation, specifically for $filter?

**Comment by jthake** [comment]
Are there particular limitations you had in mind here? This would help us understand the extra feature you are requesting...

**Comment by mishakogan** [comment]
Hi! Limitation I was talking about: delta queries only support filtering on receivedDate for messages. I wanted to filter messages with particular subject, etc... however, after reading more about deltas I realized that it might not be be the right technology for my use-case: pull emails for all users on a frequent time-basis. Delta is more for synchronizing changes and events. I have switched away from delta and now have all the filtering capabilities. Let me know if I am on the wrong path, otherwise this issue can be closed. Thanks!
